voila/app.py

```diff
@@ -348,6 +346,25 @@ def setup_template_dirs(self):
                 self.static_paths,
                 self.template_paths,
                 self.voila_configuration.template)
+            # look for possible template-related config files
+            template_conf_dir = [os.path.dirname(k) for k in self.nbconvert_template_paths]
+            conf_paths = [os.path.join(d, 'conf.json') for d in template_conf_dir]
+            for p in conf_paths:
+                # see if config path corresponds to template in use
+                if os.path.basename(os.path.dirname(p)) == self.voila_configuration.template:
```

Comment:
I don't think we should check this, actually, we'd like to 'inherit' all the configurations of the base template.

Reply:
Agreed. Fixed in fe5dd7f633dc1202b00add745d28df6a662f9781.

Reply:
Agreed. Fixed in fe5dd7f633dc1202b00add745d28df6a662f9781.